scripts/gen_hashed_bin.py

```diff
@@ -120,6 +125,18 @@ def get_args():
 		required=True, type=argparse.FileType('wb'), \
 		help='The output tee.bin')
 
+	parser.add_argument('--tee_header_v2_out', \
+		required=True, type=argparse.FileType('wb'), \
```

Comment:
Obviously, if we have several output rules as recommended above, no output parameter should be mandatory.